test/client/settings.test.jsx

```diff
@@ -16,67 +16,31 @@ const { mount } = Enzyme
 const { expect } = chai
 
 describe('Settings Component', () => {
-  let onStorageListener
-
   beforeEach(() => {
     window.localStorage.clear()
-
-    const onStorageStub = sinon.stub(window, 'onstorage')
-      .get(() => onStorageStub)
-      .set(fn => {
-        onStorageListener = fn
-      })
-
     sinon.stub(window, 'open')
   })
 
   afterEach(() => {
-    window.onstorage.restore()
     window.open.restore()
   })
 
   it('shows the sound off icon when created without sound-window key in the local storage', () => {
     const wrapper = mount(<Settings />)
 
-    expect(wrapper.find('.fas')).to.have.className('fa-volume-off')
+    expect(wrapper.find('i.icon')).to.have.className('music')
   })
 
   it('shows the sound up icon when created with sound-window key in the local storage', () => {
-    window.localStorage.setItem('sound-window', true)
```

Comment:
Why did you delete all of these tests? they are checking logic that is still exists

Reply:
No... it doesn't. The icon doesn't change anymore. Why did we use a changing icon here in the first place? The very indication of the window being open is seeing it open, there's no need for indication on the button.

Reply:
The change of the icon is not the important part, what's important is that you prevent a case of two of those windows opening.

Reply:
Ok so I changed it to only remove the tests:
 - changes the icon when sound-window appears in the local storage
 - changes the icon when sound-window removed from the local storage